packages/nx/src/generators/tree.ts

```diff
@@ -293,7 +293,10 @@ export class FsTree implements Tree {
       return !r?.isDeleted;
     });
     // Dedupe
-    return Array.from(new Set(res));
+    res = Array.from(new Set(res));
+    // in windows the result may not be sorted
+    res.sort();
+    return res;
```

Comment:
Let's revert this. Most of the time, the children don't "need" to be sorted and this could be a breaking change in some scenarios. So let's avoid it. This is a specific need for a couple of test assertions, so we can just sort the result before asserting in those particular test cases.

Reply:
done